services/galley/test/integration/API/Util.hs

```diff
@@ -1964,10 +2019,13 @@ assertMismatchQualified failedToSend missing redundant deleted = do
 
   assertExpected "deleted" deleted (fmap mssDeletedClients . responseJsonMaybe)
 
-otrRecipients :: [(UserId, [(ClientId, Text)])] -> OtrRecipients
-otrRecipients = OtrRecipients . UserClientMap . buildMap
-  where
-    buildMap = fmap Map.fromList . Map.fromList
+otrRecipients :: [(UserId, ClientId, Text)] -> OtrRecipients
```

Comment:
I forgot the slight differences between these types of user ID to ClientId to something mapping, but I remember our conclusion is that they're not quite the same. For example, the prior type that you removed on line 1967 allows for a UserId to map to an empty list, whereas the type you introduced on line 2022 (and in other places as well) does not allow it. Are you sure this change is OK?